srml/offbridge/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "offbridge"
+version = "0.1.0"
+authors = ["hammeWang <dsw0602@foxmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+serde = { version = "1.0.101", optional = true }
+safe-mix = { version = "1.0.0", default-features = false }
+codec = { package = "parity-scale-codec", version = "1.0.0", default-features = false, features = ["derive"] }
+rstd = { package = "sr-std", git = 'https://github.com/paritytech/substrate.git', default-features = false }
```

Comment:
Switch to [darwinia-network/substrate](https://github.com/darwinia-network/substrate) ?

Reply:
updated.